### PR TITLE
Fixes portals not disappearing

### LINKED ISSUE #18426: hand tele is broken and portals never fade
Could we stop with the huge amount of science bugs and general science awfulness? Playing RD is getting to be more of a hassle then fun right now. 

Hand tele portals never fade, and because they need to fade to recharge the hand tele, you can only use it three times.

### PR DESCRIPTION
**Pull request by silicons**
Fixes hand teleporter portals not disappearing
There might be a bug where the portal is created adjacent to the mob creating it instead of on it but I'm not sure, I might not be testing correctly, but that's not the point of this PR.
Fixes #18426 
:cl:
bugfix: Hand teleporters now have more then 3 uses, and their portals are no longer indefinite.
/:cl:

### CODE REVIEW

File: code/game/objects/effects/portals.dm [hunk 1]
```diff
@@ -24,13 +24,13 @@
...
 	if(A.noteleport) // No point in persisting if the target is unreachable.
+		src.visible_message("<span class='danger'>The [src] suddenly loses power and closes! The target destination is under the effects of a teleportation jammer!<span>")
 		qdel(src)
 		return
 	for(var/mob/M in src.loc)
 		src.teleport(M)
 	if(lifespan > 0)
-		spawn(lifespan)
-			qdel(src)
+		addtimer(src, "Destroy", lifespan)
```
Comment (phil235) [inline code/game/objects/effects/portals.dm]: why not qdel?

**Comment by phil235** [comment]
I don't see how that could be fixing anything.

**Comment by kevinz000** [comment]
Currently in game portals don't disappear after creation.
After I changed that line, they're disappearing fine. 
Basiclly I have no idea how but it's fixed by this.

File: code/game/objects/effects/portals.dm (hunk 1 above)
Comment (RemieRichards) [inline code/game/objects/effects/portals.dm]: Can't call ```qdel()``` here, that'd be ```thing.qdel()``` due to the way timers work, can't do that.

Should definitely be using ```qdel()``` instead though, even if it means an extra proc definition on this type that just ```qdel()```s it.

eg:
```
/portal/proc/delete()
   qdel(src)
```

**Comment by phil235** [comment]
>Basiclly I have no idea how but it's fixed by this. 

This is NOT how you do bugfixing...*sighs*

**Comment by kevinz000** [comment]
hold on i'm testing.

**Comment by kevinz000** [comment]
@phil235 properly fixed
whoever did the code forgot to indent their goddamn code right.
qdel was 1 tab too far.

**Comment by phil235** [comment]
@kevinz000 spawn() needs an indentation after it.

**Comment by PKPenguin321** [comment]
Pfft

**Comment by kevinz000** [comment]
Fuck

**Comment by kevinz000** [comment]
Ok, this bug was already fixed and I thought it wasn't because I didn't update from tg-station/master :^)
FUCK.

**Comment by Core0verload** [comment]
It wasn't, unfading portals were still possible under some conditions.

**Pull request closed without merging**